utils/issue-format-bot/lib/issueedit.js

```diff
@@ -20,6 +20,13 @@ module.exports = function(robot, alexa) {
       return;
     }
 
+    const createdAt = new Date(context.payload.issue.created_at);
+    // September 25th, 2017
+    if (createdAt.getFullYear() <= 2017 && createdAt.getMonth() <= 8 && createdAt.getDate() <= 25) {
```

Comment:
This conditional will fail for an issue created in September, 2016.

Reply:
The cleanest way to do this without a long conditional is probably formatted string comparison.  Munge `createdAt` into `YYYY-MM-DD` format, then compare it to `"2017-08-25"`

Reply:
Can't we compare `Date`s directly, or at least by conversion to timestamps?

Reply:
@Hainish Better way is to compare dates directly.

Reply:
@Hainish nice catch. Will push a fix based on https://github.com/EFForg/https-everywhere/pull/13460#discussion_r148623578